src/game.js

```diff
@@ -10,7 +10,7 @@ import { getUrl } from './assetLoader';
 import { Player } from './player';
 import { UI } from './ui/interface';
 import { Creature } from './creature';
-import dataJson from 'assets/units/data.json';
+import dataJson from './data/units.json';
```

Comment:
This file needed to be moved into `src/` so TypeScript could import it.

Reply:
Hmm, guess I'll have to eventually update https://github.com/FreezingMoon/AncientBeast-Website repo (it's using a hardcopy of the game repo itself) so that this https://ancientbeast.com/documentation/?id=units won't break with this change.

Reply:
The website itself will eventually be deprecated (there might be open issue for it) and have all the info included within the game, so we'll be showing game straight up the whole page when accessing AncientBeast.com - this will go better for embeds / arcade inclusions, though we might have to use some framework to better handle website pages and UI screens.